packages/bento-design-system/src/react-input-mask.d.ts

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "react-input-mask";
+
+declare module "react-input-mask" {
+  interface Props {
+    children: (inputProps: InputHTMLAttributes<HTMLInputElement>) => JSX.Element;
```

Comment:
should be optional?

Reply:
we use it only in one case, not sure it matters, but I can improve the type, yes